### PR TITLE
Feature/load xteas

### PR DESCRIPTION
**Pull request by Displee**

### CODE REVIEW

**Review by SchauweM — COMMENTED** [review]
Can you remove the `console.log` in run.ts since it's not needed?

File: src/filestore/stores/region-store.ts [hunk 1]
```diff
@@ -44,9 +46,20 @@ export interface Region {
 export class RegionStore {
 
     private readonly regionIndex: FileIndex;
+    private readonly xteas = {};
 
     public constructor(private fileStore: Filestore) {
         this.regionIndex = this.fileStore.getIndex('regions');
+        const array = JSON.parse(fs.readFileSync('./config/map-keys.json', 'utf8'));
```
Comment (Tynarus) [inline src/filestore/stores/region-store.ts:53]: May want to use the filestore config dir for this so the dir that the keys are stored in can be changed

**Review by Tynarus — CHANGES_REQUESTED** [review]

**Comment by Displee** [comment]
> 
> 
> Can you remove the `console.log` in run.ts since it's not needed?

You mean the console.warn?

File: src/filestore/stores/region-store.ts (hunk 1 above)
Comment (Displee) [reply src/filestore/stores/region-store.ts:53]: You mean `./unpacked`?

Comment (Tynarus) [reply src/filestore/stores/region-store.ts:53]: `filestore.configDir` :)

**Comment by Tynarus** [comment]
> > Can you remove the `console.log` in run.ts since it's not needed?
> 
> You mean the console.warn?

tbh the `run.ts` file is only for development purposes, so whatever is in it is irrelevant - it is excluded from modern published packages

**Review by Tynarus — APPROVED** [review]

**Pull request merged**